services/admin/src/store.ts

```diff
@@ -83,6 +83,14 @@ export class Store {
     return id;
   }
 
+  /**
+   * Gets all cvr file data
+   */
+  getAllCvrFiles(): string[] {
+    const rows = this.client.all('select * from cvr_files');
+    return rows.map((r) => JSON.stringify(r)).filter(Boolean);
```

Comment:
Just curious, what's the purpose of this `filter(Boolean)`?

Reply:
I actually don't know. Caroline wrote this, @carolinemodic do you remember?

Reply:
I think I copied this from some other sql query you had to filter out any null rows but it might not be necessary

Reply:
In this instance it would only filter `""`, and that isn't a possible return value of `JSON.stringify`, so it's superfluous here.

Reply:
Removed.